app/views/layouts/application.html.erb

```diff
@@ -18,6 +18,9 @@
           <li class="nav-item">
             <a class="nav-link" href="<%=users_path %>">Users</a>
           </li>
+          <li class="nav-item">
+            <a class="nav-link" href="<%=posts_path %>">Posts</a>
+          </li>
         </ul>
       </nav>
       <div class="container" style="padding-top: 5rem">
```

Comment:
Fix indentation in this file and don't use `style`, create a `.scss` create a CSS class and use it here under a `class=` clause.